### PR TITLE
Fix ssh instructions in Emergency banner docs

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/eyPmZqho/7-emergency-publishing-drill

### CODE REVIEW

File: source/manual/emergency-publishing.html.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ You can manually check whether the data has been stored in Redis by the Jenkins
 From your development machine SSH into a frontend box appropriate to the environment you want to check
 
 ```
-ssh frontend1.frontend.integration
+ssh frontend-1.frontend.integration
```
Comment (cbaines) [inline source/manual/emergency-publishing.html.md]: Well... this won't work either, given the changes to AWS?

Comment (cbaines) [reply source/manual/emergency-publishing.html.md]: Or rather, given the move to AWS.

Comment (leenagupte) [reply source/manual/emergency-publishing.html.md]: @cbaines True. I'll change the original command to point to staging and see if i can figure out how to run it on integration.

Comment (bevanloon) [reply source/manual/emergency-publishing.html.md]: @leenagupte - if you want it to point specifically to integration, one should be able to do something like 

```
ssh integration -At "ssh -q \`govuk_node_list --single-node -c frontend\`"
```

to get straight on to an integration box

Or you could go for `govukcli` https://github.com/alphagov/govuk-aws/blob/master/doc/guides/connecting-to-the-environment.md

Comment (leenagupte) [reply source/manual/emergency-publishing.html.md]: @bevanloon @cbaines I've updated the command for integration.

**Pull request merged**